compiler/aarch64/codegen/OMRInstOpCode.enum

```diff
@@ -728,6 +728,18 @@
 		vshl8h,                                                 	/* 0x4F105400,	SHL       	 */
 		vshl4s,                                                 	/* 0x4F205400,	SHL       	 */
 		vshl2d,                                                 	/* 0x4F405400,	SHL       	 */
+		vsshll_8h,                                              	/* 0x0F08A400,	SSHLL     	 */
+		vsshll_4s,                                              	/* 0x0F10A400,	SSHLL     	 */
+		vsshll_2d,                                              	/* 0x0F20A400,	SSHLL     	 */
+		vsshll2_8h,                                             	/* 0x4F08A400,	SSHLL2    	 */
+		vsshll2_4s,                                             	/* 0x4F10A400,	SSHLL2    	 */
+		vsshll2_2d,                                             	/* 0x4F20A400,	SSHLL2    	 */
+		vushll_8h,                                              	/* 0x2F08A400,	USHLL     	 */
+		vushll_4s,                                              	/* 0x2F10A400,	USHLL     	 */
+		vushll_2d,                                              	/* 0x2F20A400,	USHLL     	 */
+		vushll2_8h,                                             	/* 0x6F08A400,	USHLL2    	 */
+		vushll2_4s,                                             	/* 0x6F10A400,	USHLL2    	 */
+		vushll2_2d,                                             	/* 0x6F20A400,	USHLL2    	 */		
```

Comment:
Line endings check reports a "trailing whitespace" error at this line.

Reply:
Removed trailing whitespaces.